test/simulation/tests.go

```diff
@@ -189,7 +189,7 @@ func testOrderMatchingAndSwap(net *xudtest.NetworkHarness, ht *harnessTest) {
 		Price:    0.02,
 		Quantity: 1000000,
 		PairId:   "LTC/BTC",
-		Side:     xudrpc.OrderSide_BUY,
+		Side:     xudrpc.OrderSide_SELL,
```

Comment:
Why is this change required?

Reply:
That's an unintentional leftover, thanks for catching.